src/test/scala/io/iohk/ethereum/blockchain/sync/ScenarioSetup.scala

```diff
@@ -21,8 +21,8 @@ import scala.concurrent.ExecutionContext
   * [[io.iohk.ethereum.nodebuilder.Node Node]].
   */
 trait ScenarioSetup extends StdTestConsensusBuilder with SyncConfigBuilder with StdLedgerBuilder {
-  protected lazy val executionContext = ExecutionContext.fromExecutor(Executors.newFixedThreadPool(4))
-  protected lazy val monixScheduler = Scheduler(executionContext)
+  protected lazy val executionContextExecutor = ExecutionContext.fromExecutor(Executors.newFixedThreadPool(4))
```

Comment:
`ExecutionContext.fromExecutor` returns an `ExecutionContext` so I don't think this needs to be renamed.

Reply:
this was renamed, because the test suite extends 2 test setups, where each has an executionContext val resulting in "class BlockImporterItSpec inherits conflicting members"